packages/office-ui-fabric-react/src/components/Dialog/DialogContent.styles.ts

```diff
@@ -1,18 +1,141 @@
-import { IDialogContentStyleProps, IDialogContentStyles } from './DialogContent.types';
+import {
+  IDialogContentStyleProps,
+  IDialogContentStyles,
+  DialogType
+} from './DialogContent.types';
+import {
+  IStyle,
+  ITheme,
+  FontWeights,
+} from '../../Styling';
 
 export const getStyles = (
   props: IDialogContentStyleProps
 ): IDialogContentStyles => {
+  const {
+    className,
+    theme,
+    isLargeHeader,
+    isClose,
+    hidden,
+    isMultiline,
+  } = props;
+
+  const { palette, fonts } = theme;
+  const dialogLgHeaderBackgroundColor = palette.themePrimary;
 
   return ({
     content: [
-      'ms-DialogContent',
```

Comment:
I see that this wasn't here before, but I say leave it in.  Its an easy semantic way to find the root of this sub/sibling component in the final HTML and that could be useful - at least with any root items that have className passed to them.